src/main/java/com/excella/gradle/cucumber/CucumberRunner.java

```diff
@@ -50,7 +50,7 @@ public void runCucumberTests(JavaExec cucumberExec,
 
         if (formats != null) {
             for(String format : formats){
-                args.add("--format");
+                args.add("--plugin");
```

Comment:
It seems like this would break builds that are using older Cucumber versions, wouldn't it?